### ISSUE TITLE
Humhub 1.12: user import threws an error

### ISSUE THREAD

**Issue by optitsolutions** [humhub/massuserimport-issues#4]
Since Humhub 1.12 the mass user import tool threws an error:

Property user_id - The combination of User ID and Group ID has already been taken.
the mesaage appears for each user two times.

Environment: PHP 7.4
Humhub 1.12

**Comment by optitsolutions** [comment]
@yurabakhtin if you need further informations please ask.
the issue doesn't depend on group id's in the import file. the error appears also when the group id column is just empty.

**Comment by yurabakhtin** [comment]
@optitsolutions Thank you for the report, yes we could reproduce the error.
Fixed in PR https://github.com/humhub/massuserimport/pull/5.

**Comment by optitsolutions** [comment]
the pull section isn't visible here.
when can you publish the fix?

**Comment by yurabakhtin** [comment]
@luke- Please review the PR.

**Comment by yurabakhtin** [comment]
@optitsolutions If you need the fix urgently and can edit code, then you can add a line `$this->user->refresh();` after line `$this->user->save();` in the file `/models/UserImportContainer.php`, so new code will be:

```php
if (! $this->ignoreUserSave && $this->user->validate()) {
    $this->user->save();
    $this->user->refresh();
}
```

otherwise please wait next release, thanks.

**Comment by optitsolutions** [comment]
thank you, I'll edit the code.

**Comment by luke-** [comment]
@yurabakhtin Thanks. 

Fixed with 1.1.8
